platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -165,6 +170,14 @@ private void handleMessage(final Message msg) {
 					}
 					edit.setInputType(inputType);
 
+					if (acceptCharacters != null) {
```

Comment:
Use `TextUtils.isEmpty(acceptCharacters)` instead; the method checks for both nullity and empty strings.